### PR TITLE
first draft of healthz with three basic checks added

### PR DESCRIPTION
**Pull request by nkabz**

### CODE REVIEW

File: cmd/root.go [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+	"github.com/anmitsu/go-shlex"
+	"github.com/spf13/cobra"
+)
+
+type toCheck struct {
+	Name      string `json:"name"`
+	Type      string `json:"type"`
+	Value	  string `json:"value"`
+}
```
Comment (nkabz) [inline cmd/root.go]: @fabriciojs thinking about moving everything here to a separete "checks.go" file/class so the command itself is a more clean and lean file

File: cmd/root.go [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+	}
+
+	cmd := exec.Command(args[0], args[1:]...)
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+
+	err = cmd.Run()
+	if err != nil {
+		fmt.Printf("cmd.Run() failed with %s\n", err)
+	}
```
Comment (nkabz) [inline cmd/root.go]: those functions as well can just be a toCheck.exec(), toCheck.HTTP() and toCheck.Socket() functions, what do you think? @fabriciojs

File: cmd/root.go [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+		fmt.Printf("Problem with request: %s.\n", err.Error())
+	} else if err == nil && resp.StatusCode >= 200 && resp.StatusCode < 300 {
+		fmt.Printf("Received %d from %s\n", resp.StatusCode, url)
+	}
+}
+
+func checkExec(cmdValue string) {
+	args, err := shlex.Split(cmdValue, true)
+	if err != nil {
+		fmt.Printf("Exec args parse failed with %s\n", err)
```
Comment (nkabz) [inline cmd/root.go]: @fabriciojs Also in this first version we are simply printing values on the screen. Should we use/do something else?

Comment (nkabz) [reply cmd/root.go]: @todo here is also exit and print the exit code

I.E.: os.Exit(0) if successful and os.Exit(1) if ewe have any errors

**Review by fabriciojs — CHANGES_REQUESTED** [review]
Looking good. Waiting for the completion of test suite!

**Pull request merged**